lib/DDG/Spice/SalesTax.pm

```diff
@@ -4,41 +4,38 @@ package DDG::Spice::SalesTax;
 use strict;
 use DDG::Spice;
 use Locale::SubCountry;
-use YAML::XS 'LoadFile';
+#use YAML::XS 'LoadFile';
```

Comment:
Yes, removing it now. 
I also just realized, i don't see any cache setting. I am going to add "spice is_cached => 1;"